sdk/remoterendering/azure-mixedreality-remoterendering/azure/mixedreality/remoterendering/_remote_rendering_client.py

```diff
@@ -52,15 +54,25 @@ class RemoteRenderingClient(object):
     :param credential: Authentication for the Azure Remote
         Rendering account. Can be of the form of an AzureKeyCredential, TokenCredential or an AccessToken acquired
         from the Mixed Reality Secure Token Service (STS).
-    :type credential: Union[TokenCredential, AzureKeyCredential, AccessToken]
+    :type credential: Union[AzureKeyCredential, TokenCredential, AccessToken]
     :keyword api_version:
         The API version of the service to use for requests. It defaults to the latest service version.
         Setting to an older version may result in reduced feature compatibility.
-    :paramtype api_version: str or ~azure.mixedreality.remoterenderings.RemoteRenderingApiVersion
+    :type api_version: str or ~azure.mixedreality.remoterenderings.RemoteRenderingApiVersion
+    :keyword polling_interval:
```

Comment:
What's the type?

Reply:
Added the types here and in the async client.